tests/test_config.yaml

```diff
@@ -1,7 +1,2 @@
-lizzy-url: "https://lizzy.example/api"
-token-url: "https://auth.example/oauth2/access_token?realm=services"
-scopes: "deployer"
-user: "file_user"
-client-id: lizzy
-client-secret: 1234
-password: hunter2
+SenzaInfo:
+  - Something
```

Comment:
It is missing a `\n` in the end of this file and looks like this file belongs to `tests/fixtures/` folder.